### PR TITLE
Add Pascals triangle exercise

### PR DESCRIPTION
**Pull request by nono**
cc: @lpil

### CODE REVIEW

File: exercises/practice/pascal-triangle/.meta/example.gleam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      |> next_row()
+      |> list.prepend(acc, _)
+      |> add_rows(n - 1, _)
+  }
+}
+
+fn next_row(row: List(Int)) -> List(Int) {
+  row
+  |> list.map_fold(from: 0, with: fn(memo, item) { #(item, item + memo) })
+  |> pair.second()
```
Comment (lpil) [inline exercises/practice/pascal-triangle/.meta/example.gleam:25]:
```suggestion
  |> pair.second
```
Unlike Elixir the parens are not required here

File: exercises/practice/pascal-triangle/.meta/example.gleam [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import gleam/list
+import gleam/pair
+import gleam/result
+
+pub fn rows(n: Int) -> List(List(Int)) {
+  add_rows(n, [])
+  |> list.reverse()
```
Comment (lpil) [inline exercises/practice/pascal-triangle/.meta/example.gleam:7]:
```suggestion
  |> list.reverse
```

File: exercises/practice/pascal-triangle/.meta/example.gleam [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+fn add_rows(n: Int, acc: List(List(Int))) -> List(List(Int)) {
+  case n {
+    0 -> acc
+    _ ->
+      list.first(acc)
+      |> result.unwrap([])
+      |> next_row()
```
Comment (lpil) [inline exercises/practice/pascal-triangle/.meta/example.gleam:16]:
```suggestion
      |> next_row
```

**Review by lpil — COMMENTED** [review]
The code looks great! The build is failing, I think there is some missing config

**Comment by nono** [comment]
The build is failing because `pascals-triangle` (with an s) is not the same as `pascal-triangle` (without it)

File: exercises/practice/pascal-triangle/.meta/example.gleam (hunk 1 above)
Comment (nono) [reply exercises/practice/pascal-triangle/.meta/example.gleam:25]: It would be nice if the formatter could removes the not required parens ;-)

**Comment by nono** [comment]
And there is a conflict on `config.json` after #91 has been merged.

**Comment by nono** [comment]
It should be good now

**Review by lpil — APPROVED** [review]
Perfect! Thank you

**Pull request merged**